colors.json

```diff
@@ -1,1626 +1,1630 @@
+	},
+	"ColdFusion": {
+		"color": "#ed2cd6",
+		"url": "https://github.com/trending?l=ColdFusion"
+	},
+	"ColdFusion CFC": {
+		"color": null,
+		"url": "https://github.com/trending?l=ColdFusion-CFC"
+	},
+	"Common Lisp": {
+		"color": "#3fb68b",
+		"url": "https://github.com/trending?l=Common-Lisp"
+	},
+	"Common Workflow Language": {
+		"color": "#B5314C",
+		"url": "https://github.com/trending?l=Common-Workflow-Language"
+	},
+	"Component Pascal": {
+		"color": "#B0CE4E",
+		"url": "https://github.com/trending?l=Component-Pascal"
+	},
+	"Cool": {
+		"color": null,
+		"url": "https://github.com/trending?l=Cool"
+	},
+	"Coq": {
+		"color": null,
+		"url": "https://github.com/trending?l=Coq"
+	},
+	"Crystal": {
+		"color": "#000100",
+		"url": "https://github.com/trending?l=Crystal"
+	},
+	"Csound": {
+		"color": null,
+		"url": "https://github.com/trending?l=Csound"
+	},
+	"Csound Document": {
+		"color": null,
+		"url": "https://github.com/trending?l=Csound-Document"
+	},
+	"Csound Score": {
+		"color": null,
+		"url": "https://github.com/trending?l=Csound-Score"
+	},
+	"CSS": {
+		"color": "#563d7c",
+		"url": "https://github.com/trending?l=CSS"
+	},
+	"Cuda": {
+		"color": "#3A4E3A",
+		"url": "https://github.com/trending?l=Cuda"
+	},
+	"CWeb": {
+		"color": null,
+		"url": "https://github.com/trending?l=CWeb"
+	},
+	"Cycript": {
+		"color": null,
+		"url": "https://github.com/trending?l=Cycript"
+	},
+	"Cython": {
+		"color": null,
+		"url": "https://github.com/trending?l=Cython"
+	},
+	"D": {
+		"color": "#ba595e",
+		"url": "https://github.com/trending?l=D"
+	},
+	"Dafny": {
+		"color": "#FFEC25",
+		"url": "https://github.com/trending?l=Dafny"
+	},
+	"Dart": {
+		"color": "#00B4AB",
+		"url": "https://github.com/trending?l=Dart"
+	},
+	"DataWeave": {
+		"color": "#003a52",
+		"url": "https://github.com/trending?l=DataWeave"
+	},
+	"Dhall": {
+		"color": "#dfafff",
+		"url": "https://github.com/trending?l=Dhall"
+	},
+	"DIGITAL Command Language": {
+		"color": null,
+		"url": "https://github.com/trending?l=DIGITAL-Command-Language"
+	},
+	"DM": {
+		"color": "#447265",
+		"url": "https://github.com/trending?l=DM"
+	},
+	"Dockerfile": {
+		"color": "#384d54",
+		"url": "https://github.com/trending?l=Dockerfile"
+	},
+	"Dogescript": {
+		"color": "#cca760",
+		"url": "https://github.com/trending?l=Dogescript"
+	},
+	"DTrace": {
+		"color": null,
+		"url": "https://github.com/trending?l=DTrace"
+	},
+	"Dylan": {
+		"color": "#6c616e",
+		"url": "https://github.com/trending?l=Dylan"
+	},
+	"E": {
+		"color": "#ccce35",
+		"url": "https://github.com/trending?l=E"
+	},
+	"eC": {
+		"color": "#913960",
+		"url": "https://github.com/trending?l=eC"
+	},
+	"ECL": {
+		"color": "#8a1267",
+		"url": "https://github.com/trending?l=ECL"
+	},
+	"ECLiPSe": {
+		"color": null,
+		"url": "https://github.com/trending?l=ECLiPSe"
+	},
+	"Eiffel": {
+		"color": "#946d57",
+		"url": "https://github.com/trending?l=Eiffel"
+	},
+	"EJS": {

```

Comment:
@anmol098 I've added `EJS` key containing `color` and `URL`. My README stopped updating because of this error on `Feb 20, 2021`.